site/content/en/docs/Guides/fleet-updates.md

```diff
@@ -104,10 +105,46 @@ spec:
     - matchLabels:
         agones.dev/fleet: v2
 ```
+{{% /feature %}}
 
+{{% feature publishVersion="1.17.0" %}}
+{{< tabpane >}}
+  {{< tab header="selectors" lang="yaml" >}}
+apiVersion: "allocation.agones.dev/v1"
+kind: GameServerAllocation
+spec:
+  selectors:
+    - matchLabels:
+        agones.dev/fleet: v2
+    - matchLabels:
+        game: my-awesome-game
+  {{< /tab >}}
+  {{< tab header="deprecated required and preferred" lang="yaml" >}}
```

Comment:
Love the tabbed layout! 

![screen](https://user-images.githubusercontent.com/298370/130157135-3506b8ff-cc34-4141-8803-7da8f94841ec.png)

Nit: not sure if I'm in love with the tab titles, but I can't think of anything better 😄

Reply:
I don't like the title also but couldn't figure out a better name 😂

Reply:
Maybe: `required & preferred (deprecated)`

Just a bit more concise, and having deprecated in brackets makes it clear it's not part of the name, but is deprecated. WDYT?

Reply:
Done.